static/js/settings_account.js

```diff
@@ -97,22 +101,23 @@ exports.append_custom_profile_fields = function (element_id, user_id) {
             type = "url";
         } else if (is_user_field) {
             if (value) {
-                value = JSON.parse(value);
+                values.value = value;
             }
             type = "user";
         } else {
             blueslip.error("Undefined field type.");
         }
 
-        if (value === undefined || value === null) {
-            // If user has not set value for field.
-            value = "";
-        }
+        // if (value === undefined || value === null) {
+        //     // If user has not set value for field.
+        //     values.value = "";
+        //     values.rendered_value = "";
+        // }
```

Comment:
I assume this was just migrated up?  I'd make that a separate "preparatory refactoring" commit with its own commit message.

Reply:
Forgot to delete these lines. Sorry about that. Will fix.